acf-hide-layout.php

```diff
@@ -40,6 +40,20 @@ class ACF_Hide_Layout {
 	 */
 	protected $hidden_layouts = [];
 
+	/**
+     * File.
+     *
+     * @access protected
+     */
+    protected $file = '';
+
+    /**
+     * Basename.
+     *
+     * @access protected
+     */
+    protected $basename = '';
+
```

Comment:
Thank you for the help to remove notices. To have consistent styling these lines would have to be tabs to stay in line with WordPress coding standards.